madmom/evaluation/chords.py

```diff
@@ -720,7 +729,7 @@ class ChordEvaluation(object):
         ('undersegmentation', 'UnderSegmentation'),
     ]
 
-    def __init__(self, detections, annotations, name, **kwargs):
+    def __init__(self, detections, annotations, name='', **kwargs):
```

Comment:
I don't have a strong opinion on using `''` instead of `None`, but in the other two classes you use `None`...